src/WorkflowManager/WorkflowExecuter/Common/ArtifactMapper.cs

```diff
@@ -92,11 +106,11 @@ private static bool TrimArtifactVariable(string valueString, out string variable
             return true;
         }
 
-        private async Task<KeyValuePair<string, string>> ConvertVariableStringToPath(Artifact artifact, string variableString, string workflowInstanceId, string payloadId, string bucketId, bool shouldExistYet)
+        private async Task<KeyValuePair<string, string>> ConvertVariableStringToPath(Artifact artifact, string variableString, string workflowInstanceId, string payloadId, string bucketId, bool shouldExistYet, string suffix = "")
         {
             if (variableString.StartsWith("context.input.dicom", StringComparison.InvariantCultureIgnoreCase))
             {
-                return await VerifyExists(new KeyValuePair<string, string>(artifact.Name, $"{payloadId}/dcm/"), bucketId, shouldExistYet);
+                return await VerifyExists(new KeyValuePair<string, string>(artifact.Name, $"{payloadId}/dcm{suffix}"), bucketId, shouldExistYet);
```

Comment:
could you please also update the documentation with a few examples?

Reply:
Added some docs